### ISSUE TITLE
Proposed lightning talks for future meetups

### ISSUE THREAD

**Issue by solvingj** [orchestructure/presentations#40]
Hi All, 
Here's some simple lightning talks my team wants to work on, and part of the reason we're seeking funding for our OSS team.  They're both related to CI, automation and OSS.  Let me know if you think they're on-topic enough for the meetup.  I hope to give them a few more times as well.

https://gitpitch.com/solvingj/gitpitch-domino-lightning
https://gitpitch.com/solvingj/gitpitch-cianywhere-lightning

**Comment by StevenACoffman** [comment]
FYI, 2nd one has "aweful" and "Pmplementations". Is concourse on the radar?

**Comment by mrbobbytables** [comment]
Both of them sound like they'd be a good quick talk. Would you be interested in doing one of them at this month's meetup? (July 25th)

**Comment by solvingj** [comment]
@StevenACoffman thanks, fixed. Also, yes concourse is a very exciting project.  I would really like to see some bare-metal hosting companies like packet.net offer a SAAS implementation of the Concourse framework, like Cloudbees does for Jenkins, but fully-saas'd like Appveyor.  In most respects, it's orthagonal to CI anywhere. 

@mrbobbytables sure, would be happy to do either. Let me know if one is more interesting than the other. 

Also, I just ran through them both for the first time since I made them, and realized they're not very generalized, and some things are out of place.  They need a lot of refinement, and to assume less about the audience's background. I will improve before the 25th.
